corehq/warehouse/tests/test_staging.py

```diff
@@ -26,6 +28,7 @@ def teardown_module():
     Batch.objects.all().delete()
 
 
+@partitioned
```

Comment:
maybe we should make a BaseWarehouseTestCase so that we don't have to set the partitioned decorator on each TestCase